sci-visualization/mantid/files/mantid-3.3.0-minigzip-OF.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/MantidPlot/src/zlib123/minigzip.c b/MantidPlot/src/zlib123/minigzip.c
+index 97fbae1..5f4612f 100644
+--- a/MantidPlot/src/zlib123/minigzip.c
++++ b/MantidPlot/src/zlib123/minigzip.c
```

Comment:
After a closer look to this patches, is it possible to DEPEND on sys-libs/zlib[minizip] and build against the system lib instead of shipping an outdated zlib version and build it internally?